radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -368,26 +382,26 @@ public TokenClassReference getNativeTokenRef() {
 	 * address automatically until the observable is disposed.
 	 *
 	 * @param actionClass the Action class
-	 * @param address the address to retrieve the state of
-	 * @param <T> the Action class
+	 * @param address     the address to retrieve the state of
+	 * @param <T>         the Action class
 	 * @return a hot observable of the actions at the given address
 	 */
 	public <T> Observable<T> getActions(Class<T> actionClass, RadixAddress address) {
```

Comment:
There doesn't appear to be any meaningful changes in this method.  Why so much diff?

This really needs to be sorted out.  It's a huge waste of review time.

Reply:
I've disabled everything there is to disable on formatting actions, save actions, indent actions. I hope this will stop these annoyances. I think this is just to align the argument descriptions, but I agree, unnecessary.